SharedCode/src/commonMain/kotlin/ApplicationContract.kt

```diff
@@ -1,5 +1,6 @@
 package com.jetbrains.handson.mpp.mobile
 
+import com.jetbrains.handson.mpp.mobile.api.FaresResponse
```

Comment:
Do we need this import?

Reply:
For now I think so? It'll probably be removed when merging because of Elliot changing this part, but it's necessary in this I think